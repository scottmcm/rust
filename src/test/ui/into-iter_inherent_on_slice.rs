// run-pass

fn main() {
    for x in [1, 2][..].into_iter() {
        let _: &i32 = x;
    }
}
