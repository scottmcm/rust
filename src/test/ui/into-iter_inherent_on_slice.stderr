warning: a method with this name may be added to the standard library in the future
  --> $DIR/into-iter_inherent_on_slice.rs:4:25
   |
LL |     for x in [1, 2][..].into_iter() {
   |                         ^^^^^^^^^
   |
   = note: `#[warn(unstable_name_collisions)]` on by default
   = warning: once this method is added to the standard library, the ambiguity may cause an error or change in behavior!
   = note: for more information, see issue #48919 <https://github.com/rust-lang/rust/issues/48919>
   = help: call with fully qualified syntax `std::iter::IntoIterator::into_iter(...)` to keep using the current method
   = help: add `#![feature(you_meant_iter_not_into_iter)]` to the crate attributes to enable `core::slice::<impl [T]>::into_iter`

